spec/lib/breakers/statsd_plugin_spec.rb

```diff
@@ -53,6 +53,22 @@
 
         request.url = URI(test_host + '/v0.0/Providers(1234567890)/bar')
         expect(subject.get_tags(request)).to include('endpoint:/v0.0/xxx/bar')
+
+        request.url = URI(test_host + '/api/v1/users/b3366359af05d661342470')
+        expect(subject.get_tags(request)).to include('endpoint:/api/v1/users/xxx')
+
+        request.url = URI(test_host + '/api/v1/users/00u2sgjcthlnuo12o297')
+        expect(subject.get_tags(request)).to include('endpoint:/api/v1/users/xxx')
+
+        request.url = URI(test_host + '/api/v1/users/00u2i1p1u8m7l7FYb297/grants')
+        expect(subject.get_tags(request)).to include('endpoint:/api/v1/users/xxx/grants')
+      end
+    end
+
+    context 'request without an id' do
+      it 'doesnt replace anything' do
+        request.url = URI(test_host + '/foo/bar')
+        expect(subject.get_tags(request)).to include('endpoint:/foo/bar')
```

Comment:
Didn't need to do this, but it seems like a negative test case is appropriate to make sure that we dont accidentally filter an endpoint that we shouldn't.